docs/user/index.md

```diff
@@ -6,11 +6,11 @@ dashboard.
 
 **Prerequisites:** Podman 4.x+ OR Go version 1.18+, NodeJS version 16+, npm version 8+ 
 
-**Note:** This workshop is based on Perses v0.41.
+**Note:** This documentation is based on Perses v0.41.
```

Comment:
I think we can remove this note. Documentation should be up to date and not based on a specific version

Reply:
Fixing in my branch.